portfolio/src/main/java/com/google/sps/servlets/DeleteList.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.Arrays;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+
+/** Servlet responsible for deleting tasks. */
+@WebServlet("/delete-list")
+public class DeleteList extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query=new Query("Comment");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity commentEntity : results.asIterable()) {
+      datastore.delete(commentEntity.getKey());
+    response.sendRedirect("/comments.html");
```

Comment:
nit: this `response.sendRedirect("/comments.html");` line should be intended 2 more spaces, since it's within the for loop

Reply:
Addressed in latest commit.